hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/AbstractJpaTenantAwareBaseEntity.java

```diff
@@ -32,7 +31,6 @@
     private static final long serialVersionUID = 1L;
 
     @Column(name = "tenant", nullable = false, insertable = false, updatable = false, length = 40)
-    @Size(max = 40)
     private String tenant;
```

Comment:
Because tenant cannot be set by a user customers. A java validation is not required imho.